common/Taskfile.yml

```diff
@@ -1,5 +1,11 @@
 version : '3'
 
+includes:
+  infra:
+    dir: "{{ .TASKFILE_DIR }}"
+    taskfile: infrastructure-tooling/Infra.yml
```

Comment:
I feel like it would be better to check out `infrastructure-tooling` at the same level as `packages`.

Then it would be
`taskfile: ../infrastructure-tooling/Infra.yml`

Reply:
I assume you meant root of the clone and then taskfile: ./infrastructure-tooling/Infra.yml

instead of packages/packages/ taskfile: ../infrastructure-tooling/Infra.yml

If I am wrong I will need further instruction because it was not obeying me when I trying to checkout from packages/packages and source up.

Reply:
I think @ReillyBrogan meant this directory structure:
```
.
├── infrastructure-tooling
│   └── Infra.yml
└── packages
    ├── common
    ├── packages
    ├── README.md
    └── Taskfile.yml -> common/Taskfile.yml
```

With the include being:

```yaml
includes:
  infra:
    dir: ../infrastructure-tooling
    taskfile: ../infrastructure-tooling/Infra.yml
    optional: true
```

And `Infra.yml` using `TASKFILE_DIR` instead of `ROOT_DIR` (see [special variables](https://taskfile.dev/api/#special-variables)).

Reply:
Requested changes made including submitting PR for Infra.yml